packages/client/src/subscribe/Subscriber.ts

```diff
@@ -35,6 +38,58 @@ export default class Subscriber implements Context {
         return this.subscribeTo(streamPartId, onMessage)
     }
 
+    /**
+     * Subscribe to all partitions for stream.
+     */
+    async subscribeAll<T>(streamId: StreamID, onMessage?: SubscriptionOnMessage<T>): Promise<MessageStream<T>> {
```

Comment:
Will the user be able to unsubscribe this subscription by calling `sub.unsubcribe()`? 

Maybe this could return a `Subscription` instead of `MessageStream`, if we convert `Subscription` to a interface (having just `unsubscribe()` methods)?

Reply:
Moved `unsubscribe` onto the `MessageStream`.

Reply:
What would be the role of `Subscription` class when this PR is in `main`? Will also the subscribe() method returns `MessageStream` and then we remove the `Subscription` class?